chapter01/chapter01.adoc

```diff
@@ -0,0 +1,263 @@
+
+[[eclipse-foundation-working-group]]
+Eclipse Foundation Working Group:: A collaborative group of industry leaders and Java community members who actively contribute to the of development of Eclipse projects like MicroProfile within the Eclipse Foundation framework.
+
+[[jakarta-ee]]
+Jakarta EE:: Jakarta EE (formerly Java Platform, Enterprise Edition, or Java EE) is a set of specifications, extending Java Platform, Standard Edition, or Java SE with specifications for enterprise features such as web services, database persistence, asynchronous messaging and more.
+
+[[external-configuration]]
+External Configuration:: A technique in application development where configuration data is separated from the application code, allowing the application's behavior to be adjusted without changing the code, especially useful in cloud-native and microservices architectures.
+
+[[health-checks]]
+Health Checks:: Mechanisms used in microservices architectures to continuously check the status of an application or service to ensure it is functioning correctly and available to users.
+
+[[fault-tolerance]]
+Fault Tolerance:: The ability of a system to continue operating in the event of the failure of some of its components. This feature is critical for maintaining high availability and reliability in microservices architectures.
+
+[[vendor-neutrality]]
+Vendor Neutrality:: The principle of designing software products and standards not controlled by any single vendor, promoting user interoperability and choice.
+
+[[interoperability]]
+Interoperability:: The ability of a software to exchange and make use of information across different platforms and services.
+
+[[json-p]]
+JSON-P (JSON Processing):: A Java API that enables parsing, generating, transforming, and querying JSON data. It facilitates the processing of JSON data within the Java programming environment.
+
+[[json-b]]
+JSON-B (JSON Binding):: A Java API for binding Java objects to JSON messages and vice versa, streamlining the serialization and deserialization process. It allows custom mappings to handle complex conversion scenarios efficiently.
+
+[[jax-rs]]
+JAX-RS (Java API for RESTful Web Services):: It supports creating web services according to the REST architectural pattern in Java, using annotations to simplify development. It enables the easy creation and management of resources via standard HTTP methods.
+
+[[cdi]]
+CDI (Contexts and Dependency Injection):: A set of Java services and APIs for enterprise-grade dependency injection, offering type-safe mechanisms, context lifecycle management, and a framework for decoupling application components. It enhances modularity and facilitates the development of loosely coupled, easily testable applications.
```

Comment:
Change after the double colon: A Jakarta EE API that provides services and APIs for enterprise-grade ...

Reply:
I suggest you add CDI Lite as it was included in Jakarta EE 10 Core Profile, which is the one MP 6.1 supports.

Reply:
@edbratt - would change to Jakarta EE API
@Emily-Jiang Have mentioned about CDI Lite in Core Profile Section, In this chapter we are taking about inception of MicroProfile. Thats why using CDI.